100_essentials/messages/README.md

```diff
@@ -0,0 +1,42 @@
+# Messages
+
+You've written a bunch of messages but you realize not that your shift key was broken. Write a function that Capitalizes the first letter of a message.
```

Comment:
Grammar note:

> but you realize not that your shift key was broken

Should likely be

> but you failed to realize that your shift key was broken.

Or

> but you didn't realize that your shift key was broken.

---

Definitely nitpicky, but I think it would add some clarity to the initial challenge statement. I'm not going to make it a blocker though.